Headers/QuartzCore/CALayer.h

```diff
@@ -30,34 +30,34 @@
 #import "QuartzCore/CABase.h"
 #import "QuartzCore/CAMediaTiming.h"
 #import "QuartzCore/CATransform3D.h"
-#if GNUSTEP
+//#if GNUSTEP
 #import <CoreGraphics/CoreGraphics.h>
-#endif
-
-NSString *const kCAGravityResize;
-NSString *const kCAGravityResizeAspect;
-NSString *const kCAGravityResizeAspectFill;
-NSString *const kCAGravityCenter;
-NSString *const kCAGravityTop;
-NSString *const kCAGravityBottom;
-NSString *const kCAGravityLeft;
-NSString *const kCAGravityRight;
-NSString *const kCAGravityTopLeft;
-NSString *const kCAGravityTopRight;
-NSString *const kCAGravityBottomLeft;
-NSString *const kCAGravityBottomRight;
-NSString *const kCATransition;
+//#endif
+
+extern NSString *const kCAGravityResize;
+extern NSString *const kCAGravityResizeAspect;
+extern NSString *const kCAGravityResizeAspectFill;
+extern NSString *const kCAGravityCenter;
+extern NSString *const kCAGravityTop;
+extern NSString *const kCAGravityBottom;
+extern NSString *const kCAGravityLeft;
+extern NSString *const kCAGravityRight;
+extern NSString *const kCAGravityTopLeft;
+extern NSString *const kCAGravityTopRight;
+extern NSString *const kCAGravityBottomLeft;
+extern NSString *const kCAGravityBottomRight;
+extern NSString *const kCATransition;
 
 @class CAAnimation;
 @class CARenderer;
 
 #import "QuartzCore/CAAction.h"
 
-#if GNUSTEP
+//#if GNUSTEP
```

Comment:
Could you please explain why you had to change this? The old code is a horrible workaround, but using it everywhere looks even worse.